packages/react-core/src/components/ClipboardCopy/ClipboardCopyExpanded.tsx

```diff
@@ -6,7 +6,6 @@ import { PickOptional } from '../../helpers/typeUtils';
 
 export interface ClipboardCopyExpandedProps extends Omit<ClipboardCopyProps, 'onChange'> {
   className?: string;
-  children: React.ReactNode;
```

Comment:
Did you mean to remove this?

Reply:
yep - it matches ClilpboardCopyProps now

Reply:
Since the type is changing , we can remove the array logic.  However I wonder if this would break people who using the component like mentioned in the issue that resulted in adding array logic. https://github.com/patternfly/patternfly-react/issues/6863